tests/python/unittest/test_meta_schedule_task_scheduler.py

```diff
@@ -337,6 +337,7 @@ def test_meta_schedule_task_scheduler_override_next_task_id_only():  # pylint: d
         )
 
 
+@pytest.mark.skip("Does array access OOB")
```

Comment:
Can you file and link an issue here with the context? Also it'd be nice if this was a `skipIf`, maybe you can use a combo of flags from libinfo https://github.com/apache/tvm/blob/main/src/support/libinfo.cc#L238?